### DISCUSSION TITLE
Solution accuracy

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2233] [General]
When I run 1000 cycles, several warnings appear during the solution. 
For example: psetup failed: .../casadi/interfaces/sundials/idas_interface.cpp:849: Calculating Jacobian failed; At t = 0.00844707 and h = 2.17202e-18, the corrector convergence failed repeatedly or with |h| = hmin, Will this affect the accuracy of my final solution results?

**Comment by valentinsulzer** [comment]
No, it will not affect the accuracy

**Reply by RobinBluehub** [reply]
What about this warning: Linear solve failed，does it affect the accuracy of the solution?

**Reply by valentinsulzer** [reply]
No. In both cases the solver tried to take steps that are too large and failed, but later succeeded with a smaller step. If you decrease both the period of the experiment and the `dt_max` attribute, those warnings might go away (but it might also be slower)

**Reply by RobinBluehub** [reply]
thank you.
